platform-sdk/swirlds-common/src/main/java/com/swirlds/common/context/PlatformContext.java

```diff
@@ -53,28 +54,37 @@ public interface PlatformContext {
     static PlatformContext create(@NonNull final Configuration configuration) {
         final Metrics metrics = new NoOpMetrics();
         final Cryptography cryptography = CryptographyHolder.get();
-        return create(configuration, metrics, cryptography);
+        // This new method is used by commands that have no access to nodeId,
+        // we temporary assigning node 0 until we remove recycle bin from the fsManager,
+        // then we can set a dummy impl of the recycle bin
+        final FileSystemManager fileSystemManager =
+                FileSystemManagerFactory.getInstance().createFileSystemManager(configuration, metrics);
+
+        return create(configuration, metrics, cryptography, fileSystemManager);
```

Comment:
Assuming node is going to make local testing tricky (currently `pcli browse` platforms share the same file system). It is reasonable for the platform context to be aware of the node ID, this should be an argument to whatever builds the platform context.

I started work on a platform context builder (in draft mode because there is ongoing design discussion around the platform context), and in that PR the node ID was required to build a platform context. https://github.com/hashgraph/hedera-services/pull/13152

Reply:
As spoken, fixed the comment and moved it to javadoc.
It is a method only used by commands not aware of the concept of nodeId.